CRM/Contact/BAO/Query.php

```diff
@@ -6355,15 +6355,14 @@ public static function processSpecialFormValue(&$formValues, $specialFields, $ch
    * @param string|CRM_Utils_Sort $sort
    *   The order by string.
    * @param null $sortOrder
-   *   Who knows? Hu knows. He who knows Hu knows who.
+   *   Who knows? Hu knows. One who knows Hu knows who.
```

Comment:
I feel like this formulation needs the word "won" as well.

Reply:
Hu knows Won?

Reply:
If Won fought Hu... who won?